upgrade/scripts/upgrade/csm-upgrade.sh

```diff
@@ -156,6 +156,20 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+state_name="UNINSTALL_ETCD_OPERATOR"
+#shellcheck disable=SC2046
+state_recorded=$(is_state_recorded "${state_name}" $(hostname))
+if [[ $state_recorded == "0" ]]; then
+    echo "====> ${state_name} ..."
+    {
+    helm uninstall -n operators cray-etcd-operator
```

Comment:
Judging by a quick skim of the ticket, this seems to only be needed in Kubernetes 1.22.

Since this may run in 1.5 alpha or beta, can this instead determine if it should run based on the current Kubernetes version?